docs/manuals/cli.md

```diff
@@ -0,0 +1,245 @@
+By installing Taipy, you have access to the `taipy` command-line. Several commands are available.
+
+# Get Taipy version
+You can check your current version of Taipy by running the `taipy --version` command in a terminal
+(Linux, macOS) or command prompt (Windows).
+
+```console
+$ taipy --version
+Taipy 2.3.0
+```
+
+If you don't see a supported version of Taipy, you'll need to either upgrade Taipy or perform a
+fresh install, as described in the [Installation page](../installation.md).
+
+# Help on Taipy CLI
+A list of available commands provided by Taipy is shown by running the `taipy`, or `taipy help`,
+or `taipy --help` command.
+
+```console
+$ taipy
+usage: taipy [-v] {manage-versions,create,help} ...
+
+positional arguments:
+  {manage-versions,create,help}
+    manage-versions     Taipy version control system.
+    create              Create a new Taipy application.
+    help                Show the Taipy help message.
+
+options:
+  -v, --version         Print the current Taipy version and exit.
+```
+
+# Create a Taipy application
```

Comment:
Find a better name:
Create ?

Reply:
I like "Create a Taipy application" better, because when pairing with the subsection title, it will form a full sentence.

"Create" is really not clear.